backend/inspirehep/matcher/api.py

```diff
@@ -239,3 +242,62 @@ def get_affiliations_from_pdf(url, **kwargs):
     if not response:
         return
     return {"authors": parsed_authors}
+
+
+def exact_match_workflow_data(data):
+    """Return matched record ids if the record is already present in the system.
+
+    Uses the default configuration of the ``inspire-matcher`` to find
+    duplicates of the current workflow object in the system.
+
+    Also sets the ``matches.exact`` property in ``extra_data`` to the list of
+    control numbers that matched.
+
+    Arguments:
+        data: data used for matching.
+
+    Returns:
+        List: List of matched ids
+
+    """
+    exact_match_workflow_matcher_config = current_app.config[
+        "EXACT_WORKFLOW_MATCH_CONFIG"
```

Comment:
Let's rename everything with workflow to generic names, we might use it in other cases too.

Reply:
Well, I am not sure, from the one hand I know what you mean (hep is a separate service), but on the other hand, if we don't add a prefix it might be confusing - exact match or fuzzy match what (which index)?

Reply:
hmm I see, what if we  do something like `EXACT_LITERATURE_MATCH_CONFIG`?